mart/configs/attack/classification_fgsm_linf.yaml

```diff
@@ -6,6 +6,7 @@ defaults:
   - gradient_modifier: sign
   - gain: cross_entropy
   - objective: misclassification
+  - batch_c15n: list
```

Comment:
why is this not `tuple`?

Reply:
It shouldn't really matter, but `torch.utils.data.Dataloader` returns a `list` instead of `tuple` by default: https://github.com/pytorch/pytorch/blob/9d2ffc5dfa783cf13f8cb49e6cef05d3c6335e76/torch/utils/data/_utils/collate.py#L71

However, we wrote a custom `collate_fn()` for coco, which gives `tuple`. That's why the difference.
https://github.com/IntelLabs/MART/blob/df1734fdbcec9c318575d45c3396f44fbba38959/mart/datamodules/coco.py#L74